gui/default/index.html

```diff
@@ -522,12 +522,22 @@ <h4 class="panel-title">
-                          <span ng-switch-when="trashcan" translate>Trash Can File Versioning</span>
-                          <span ng-switch-when="staggered" translate>Staggered File Versioning</span>
-                          <span ng-switch-when="simple" translate>Simple File Versioning</span>
-                          <span ng-switch-when="external" translate>External File Versioning</span>
+                        <th><span class="fa fa-fw fa-file"></span>&nbsp;<span translate>File Versioning</span></th>
+                        <td class="text-right">
+                          <span ng-switch="folder.versioning.type">
+                            <span ng-switch-when="trashcan" translate>Trash Can</span>
+                            <span ng-switch-when="simple" translate>Simple</span>
+                            <span ng-switch-when="staggered" translate>Staggered</span>
+                            <span ng-switch-when="external" tooltip data-original-title="<span class='text-monospace'>{{folder.versioning.params.command}}</span>" translate>External</span>
+                          </span>
+                          <span ng-if="folder.versioning.type != 'external'">
+                            <span ng-if="(folder.versioning.type == 'trashcan' || folder.versioning.type == 'simple') && folder.versioning.params.cleanoutDays != versioningDefaults.trashcanClean" tooltip data-original-title="{{'Clean out after' | translate}}">&ensp;<span class="fa fa-calendar"></span>&nbsp;{{folder.versioning.params.cleanoutDays * 86400 | duration:"d"}}</span>
+                            <span ng-if="folder.versioning.type == 'simple' && folder.versioning.params.keep != versioningDefaults.simpleKeep" tooltip data-original-title="{{'Keep Versions' | translate}}">&ensp;<span class="fa fa-file-archive-o"></span>&nbsp;{{folder.versioning.params.keep}}</span>
+                            <span ng-if="folder.versioning.type == 'staggered' && folder.versioning.params.maxAge != versioningDefaults.staggeredMaxAge" tooltip data-original-title="{{'Maximum Age' | translate}}">&ensp;<span class="fa fa-calendar"></span>&nbsp;{{folder.versioning.params.maxAge | duration}}</span>
+                            <span ng-if="folder.versioning.cleanupIntervalS != versioningDefaults.cleanupIntervalS" tooltip data-original-title="{{'Cleanup Interval' | translate}}">&ensp;<span class="fa fa-recycle"></span>&nbsp;<span ng-if="folder.versioning.cleanupIntervalS == 0" translate>Disabled</span><span ng-if="folder.versioning.cleanupIntervalS > 0">{{folder.versioning.cleanupIntervalS | duration}}</span></span>
+                            <!-- Keep the path last, so that it truncates without pushing other information out of the screen. -->
+                            <span ng-if="folder.versioning.fsPath != ''" tooltip data-original-title="{{folder.versioning.fsPath}}">&ensp;<span class="fa fa-folder-open-o"></span>&nbsp;{{folder.versioning.fsPath}}</span>
```

Comment:
These lines are ultra-long, might be worthwhile to add newlines to put the opening and closing span tags and the content between on separate lines.

Reply:
Thank you for having a look at the PR 🙂.

> Will likely be clunky on smalls screens, but the view already is.

It's not bad, actually. The information is just moved to the second line as below.

![image](https://user-images.githubusercontent.com/5626656/170673849-1de3cdc8-2c07-4815-8268-a4a7b6f60a6f.png)

Reply:
I meant the second one. Just a suggestion though - if you agree that it makes sense that away, go with it, otherwise it's also fine as is for me.

Reply:
It surely does make sense 🙂. The lines have been separated.